### DISCUSSION TITLE
symfony/cache 版本问题，失踪默认安装5.4x 然后就是这个bug

### DISCUSSION THREAD

**Discussion by zeroone2005** [w7corp/easywechat#2793] [Q&A]
psr/simple-cache 3.0+ is not compatible with this version of symfony/cache. Please upgrade symfony/cache to 6.0+ or downgrade psr/simple-cache to 1.x or 2.x.

**Comment by baiwapak** [comment]
我也遇到这个问题，求大佬们提供思路

**Reply by jrphp** [reply]
大佬们，解决了吗？我也遇到这个问题了

**Reply by xing69164784** [reply]
升级 composer require symfony/cache:^6.0 -W     
或降级 composer require psr/simple-cache:^2.0 -W

**Reply by zimuyun** [reply]
> 升级 composer require symfony/cache:^6.0 -W 或降级 composer require psr/simple-cache:^2.0 -W

升级不行，降级composer require psr/simple-cache:^2.0 -W 可以，php8.3

**Comment by overtrue** [comment]
安装哪个版本是根据各种环境算出来的，PHP 版本，其他包的支持情况等。

**Comment by overtrue** [comment]
可以通过这个命令查看它被哪些库依赖，以及对应的版本：

```bash
composer why symfony/cache
```
